runtime/appruntime/config/config.go

```diff
@@ -81,6 +84,12 @@ type CORS struct {
 	// the default set of {"Origin", "Authorization", "Content-Type"}.
 	// As a special case, if the list contains "*" all headers are allowed.
 	ExtraAllowedHeaders []string `json:"raw_allowed_headers,omitempty"`
+
+	// AllowAccessWhenOnPrivateNetwork, if true, allows requests to Encore apps running
+	// on private networks from websites.
+	//
+	// See: https://wicg.github.io/private-network-access/
+	AllowAccessWhenOnPrivateNetwork bool `json:"allow_private_network_access,omitempty"`
```

Comment:
The field name is different from the JSON key, make them consistent? I think the JSON field is clearer